api/r/cellxgene.census/vignettes_/census_dataset_presence.Rmd

```diff
@@ -40,7 +40,8 @@ Contents
 The `cellxgene.census` R package contains a convenient API to open any version of the Census (by default, the newest stable version).
 
 ```{r}
-census <- cellxgene.census::open_soma()
+library("cellxgene.census")
```

Comment:
I don't feel strongly about this, but the quotes aren't required and not usually used IME. (Throughout)

```suggestion
library(cellxgene.census)
```

Reply:
I think there's been a shift. Mainly because if the package is not installed you get an variable not found error instead of a package not installed error